app/views/mentee_application_mailer/notify_for_reapplication.html.erb

```diff
@@ -0,0 +1,57 @@
+
+<div style='display: block; max-width: 40rem; margin: 0 auto; padding-top: 4rem; line-height: 1.3rem; padding: 0.25rem'>
+  <p>Dear <%= @recipient.full_name %>,</p>
+  <p>
+    Thank you for previously expressing interest to join us at the Agency of Learning.
+  </p>
+
+  <p>
+    We are currently opening our upcoming <%= @active_cohort_name %> cohort to early-career developers with some experience using Ruby on Rails to join us. In particular, we are looking for applicants residing in North America who can devote at least 10 hours per week to the Agency.
+  </p>
+
+  <p>
+    Given your previous interest, we wanted to invite you to <strong>reapply to the Agency of Learning</strong>. To speed things up, we've filled in your reapplication form with information from your previous application.
+  </p>
+
+  <p>
+    To reapply, please follow these steps:
+  </p>
+
+  <h4>Reset your password</h4>
+  <ol>
+    <li>
+      Navigate to <a href="https://pair.agencyoflearning.com/users/sign_in">the agency's pair application sign in form</a>
```

Comment:
So I guess this mailer will just be run once and then removed or changed? If it's going to stick around, I think it's probably better to use url helpers instead of hardcoding the prod urls in.

Reply:
Will change to use url helpers, good call out.